### DISCUSSION TITLE
Speedometer

### DISCUSSION THREAD

**Discussion by adrian-bender** [jspsych/jsPsych#1727] [General]
Hi,

m kind of newbie in this incredible tool. I have to copy a trial which in its original version ask the user to input a continuous value through a click on an image like this.

![image](https://user-images.githubusercontent.com/13291773/114629296-b9b70300-9c8e-11eb-8f86-6e96e64c1da9.png)

Is it possible to implement a control like this? How difficult is it? Can you guide me on how to implement it?

**Comment by becky-gilbert** [comment]
Hi @adrian-bender, I developed a jsPsych 'dial response' plugin, which I think might do what you need: [jspsych-dial-response repo](https://github.com/becky-gilbert/jspsych-dial-response). It uses the jQuery-Knob library. You can download the code from the repository and run the [example HTML file](https://github.com/becky-gilbert/jspsych-dial-response/blob/master/jspsych-dial-response.html) locally, or see the live demo on cognition.run here: https://kb8n5md7ql.cognition.run/ I think you'd want some combination of the 3rd example trial (angle_arc: 180) and 5th example trial (cursor_mode: false, thickness: 1). I didn't write any documentation, but the parameters and descriptions are all listed in the [plugin fie](https://github.com/becky-gilbert/jspsych-dial-response/blob/master/jspsych-dial-response.js).

**Comment by adrian-bender** [comment]
This is really helpful. Thank you very much Becky.
